SharePointFramework/PortfolioWebParts/src/webparts/projectList/manifest.json

```diff
@@ -27,7 +27,12 @@
                 "sortBy": "Title",
                 "showAsTiles": true,
                 "showProjectOwner": true,
-                "showProjectManager": true
+                "showProjectManager": true,
+                "showProjectStatus": true,
+                "showLifeCycleStatus": true,
+                "showServiceArea": true,
+                "showType": true,
+                "termSetIds": "abcfc9d9-a263-4abb-8234-be973c46258a"
```

Comment:
`termSetIds` should be `termSetId` if it's not plural.

Reply:
This could be plural if any customer choose to use more than the regular termSet for phaseLevels